server/controllers/events.js

```diff
@@ -0,0 +1,32 @@
+const models = require('../../db/models');
+
+module.exports.create = (req, res) => {
+  console.log(req.body);
+  models.Event.forge({title: req.body.eventName, creator_id: req.session.passport.user})
```

Comment:
No, it refers to the creator_id column in the database. It has to be creator_id.